puttputt/views.py

```diff
@@ -54,12 +64,188 @@ def register(request):
 def index(request):
     all_members = User.objects.all
 
+    profile = None
+    player_info = None
+
+    tournament = None
+
+    if request.user.is_authenticated:
+        info = get_player(request.user)
+        profile = info['profile']
+        player_info = info['player_info']
+        tournament = player_info.tournament
+    else: 
+        print('no authorized user')
+
+    #if profile != None and player_info != None:
+        #tournament_for_player(player_info)
+
+    context = {'all': all_members, 'profile': profile, 'player_info' : player_info, 'tournament' : tournament}
+
     return render(request,
     'puttputt/index.html',
-    {'all': all_members}
-)
+    context)
 
 def logout_request(request):
     logout(request)
     return redirect("index")
 
+# the main game 
+def game(request):
+    game_info = get_game_info(request.user)
+
+    score = game_info['current_score']
+    hole = game_info['current_hole']
+
+    last_hole = hole == VenueInfo.objects.all()[:1].get().number_of_holes
+
+    context = {'score': score, 'hole': hole, 'last_hole': last_hole}
+
+    return render(request, 'puttputt/game.html', context)
+
+# this function serves the purpose of eating game stroke submissions
```

Comment:
Turn this comment into a docstring:
```suggestion
def add_stroke(request):
    """this function serves the purpose of eating game stroke submissions
    then redirecting back to the game board so they can't accidentally refresh
    then submit again
    """
    print('adding a stroke')
```